app/pages/cliniciandetails/cliniciandetails.js

```diff
@@ -38,52 +40,68 @@ class ClinicianDetails extends React.Component {
         firstName: this.getUserFirstName(),
         lastName: this.getUserLastName(),
         clinicalRole: '',
+        country: '',
       },
       validationErrors: {},
-    }
+    };
     this.handleSubmit = this.handleSubmit.bind(this);
     this.handleInputChange = this.handleInputChange.bind(this);
   }
 
   formInputs() {
-    return [
-    {
-      name: 'firstName',
-      label: t('First Name'),
-      type: 'text',
-      placeholder: t('First name')
-    },
-    {
-      name: 'lastName',
-      label: t('Last Name'),
-      type: 'text',
-      placeholder: t('Last name')
-    },
-    {
-      name: 'clinicalRole',
-      label: t('Clinical Role'),
-      type: 'select',
-      value: '',
-      placeholder: t('Select Role...'),
-      items: [
-        {value: 'clinic_manager', label: t('Clinic Manager')},
-        {value: 'diabetes_educator', label: t('Diabetes Educator')},
-        {value: 'endocrinologist', label: t('Endocrinologist')},
-        {value: 'front_desk', label: t('Front Desk')},
-        {value: 'information_technology', label: t('IT/Technology')},
-        {value: 'medical_assistant', label: t('Medical Assistant')},
-        {value: 'nurse', label: t('Nurse/Nurse Practitioner')},
-        {value: 'primary_care_physician', label: t('Primary Care Physician')},
-        {value: 'physician_assistant', label: t('Physician Assistant')},
-        {value: 'other', label: t('Other')}
-      ]
+    const inputs = [
+      {
+        name: 'firstName',
+        label: t('First Name'),
+        type: 'text',
+        placeholder: t('First name')
+      },
+      {
+        name: 'lastName',
+        label: t('Last Name'),
+        type: 'text',
+        placeholder: t('Last name')
+      },
+      {
+        name: 'clinicalRole',
+        label: t('Clinical Role'),
+        type: 'select',
+        value: '',
+        placeholder: t('Select Role...'),
+        items: [
+          { value: 'clinic_manager', label: t('Clinic Manager') },
+          { value: 'diabetes_educator', label: t('Diabetes Educator') },
+          { value: 'endocrinologist', label: t('Endocrinologist') },
+          { value: 'front_desk', label: t('Front Desk') },
+          { value: 'information_technology', label: t('IT/Technology') },
+          { value: 'medical_assistant', label: t('Medical Assistant') },
+          { value: 'nurse', label: t('Nurse/Nurse Practitioner') },
+          { value: 'primary_care_physician', label: t('Primary Care Physician') },
+          { value: 'physician_assistant', label: t('Physician Assistant') },
+          { value: 'other', label: t('Other') }
+        ]
+      },
+    ];
+    if (config.ALLOW_SELECT_COUNTRY) {
+      inputs.push({
+        name: 'country',
+        label: t('Country'),
+        type: 'select',
+        placeholder: t('Select Country...'),
+        value: '',
+        items: [
+          { value: 'FR', label: t('France') },
```

Comment:
Could we make this part of configuration as for the languages? Otherwise each time we add a new country we'll have to change the code.
Perhaps reuse the file languages.json. Or create a dedicated country.json to separate language from country...